### PR TITLE
Pet Store Api Testing

### PR DESCRIPTION
**Pull request by CarlosGuadamuz0**

### CODE REVIEW

File: src/main/java/json_homework/json_reader/JsonReader.java [hunk 1]
```diff
@@ -1,16 +1,23 @@
 package json_homework.json_reader;
 
-import com.google.gson.Gson;
+import com.google.gson.*;
```
Comment (BohdanKalep) [inline src/main/java/json_homework/json_reader/JsonReader.java:3]: Update import

File: src/main/java/json_homework/json_reader/JsonReader.java [hunk 2]
```diff
@@ -48,4 +55,206 @@ public static String readFile(String filePath) throws IOException {
...
+            }
+        });
+
+        return filledTemplate;
+    }
+
+    /**
+     * Convert a Json Object into an url encoded data
+     */
+    public static String convertJsonObjectToUrlEncoded(JsonObject jsonObject) {
```
Comment (BohdanKalep) [inline src/main/java/json_homework/json_reader/JsonReader.java:160]: Don't use url encoder to encode json body

File: src/main/java/json_homework/json_reader/JsonReader.java [hunk 3]
```diff
@@ -48,4 +55,206 @@ public static String readFile(String filePath) throws IOException {
...
+    }
+
+    /**
+     * Returns a JsonObject[]  using the template to fill the data
+     */
+    public static JsonObject[] readTestDataAndBuild(String testDataFile, String templateFile, String templateName, String testDataKey) throws IOException {
+        try (Reader dataReader = new FileReader(testDataFile);
+             Reader templateReader = new FileReader(templateFile)) {
+
+            JsonArray testDataObject = JsonParser.parseReader(dataReader).getAsJsonObject()
```
Comment (BohdanKalep) [inline src/main/java/json_homework/json_reader/JsonReader.java:181]: Where is catch block?

File: src/main/java/json_homework/json_reader/JsonReader.java [hunk 4]
```diff
@@ -48,4 +55,206 @@ public static String readFile(String filePath) throws IOException {
...
+                        throw new RuntimeException(e);
+                    }
+                })
+                .collect(Collectors.joining("&"));
+    }
+
+    /**
+     * Returns a JsonObject[]  using the template to fill the data
+     */
+    public static JsonObject[] readTestDataAndBuild(String testDataFile, String templateFile, String templateName, String testDataKey) throws IOException {
```
Comment (BohdanKalep) [inline src/main/java/json_homework/json_reader/JsonReader.java:177]: Do not throws exception if you handle it in the method

File: src/main/java/json_homework/json_reader/JsonReader.java [hunk 5]
```diff
@@ -48,4 +55,206 @@ public static String readFile(String filePath) throws IOException {
...
+        }
+    }
+
+    /**
+     * Returns a JsonArray[] using the template to fill the data
+     */
+    public static JsonArray[] readTestDataAndBuildArray(String testDataFile, String templateFile, String templateName, String testDataKey) throws IOException {
+        try (Reader dataReader = new FileReader(testDataFile);
+             Reader templateReader = new FileReader(templateFile)) {
+
```
Comment (BohdanKalep) [inline src/main/java/json_homework/json_reader/JsonReader.java:198]: Where is catch block?

File: src/test/java/api_constants/ApiTestConstants.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
+package api_constants;
+
+public class ApiTestConstants {
+    public static final String CONTENT_TYPE_JSON = "application/json";
+    public static final String CONTENT_TYPE_FORM = "application/x-www-form-urlencoded";
+    public static final int SC_OK = 200;
+    public static final int SC_NOT_FOUND = 404;
```
Comment (BohdanKalep) [inline src/test/java/api_constants/ApiTestConstants.java:7]: Apache is providing you this status codes. You don't need to create them

File: src/test/java/api_constants/ApiTestConstants.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    public static final int SC_UNSUPPORTED_MEDIA_TYPE = 415;
+    public static final int SC_INTERNAL_SERVER_ERROR = 500;
+    public static final String CODE = "code";
+    public static final String EMPTY_ARRAY = "[]";
+    public static final String BASE_PATH = "src/test/java/";
```
Comment (BohdanKalep) [inline src/test/java/api_constants/ApiTestConstants.java:12]: Create PathHandler and store there everything related to path

File: src/test/java/petstore/PetstoreConstants.java [hunk 8]
```diff
@@ -0,0 +1,6 @@
+package petstore;
+
+public class PetstoreConstants {
+    public static final String BASE_URL = "https://petstore.swagger.io/v2";
```
Comment (BohdanKalep) [inline src/test/java/petstore/PetstoreConstants.java:4]: Make it a URL builder and keep here everything related to url creation

File: src/test/java/petstore/pet/PetApiTest.java [hunk 9]
```diff
@@ -0,0 +1,308 @@
...
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+import petstore.pet.data.PetApiData;
+
+
+import java.io.File;
+import java.util.Objects;
+
+
+import static api_constants.ApiTestConstants.*;
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:15]: Remove wildcards

File: src/test/java/petstore/pet/PetApiTest.java [hunk 10]
```diff
@@ -0,0 +1,308 @@
...
+import static io.restassured.RestAssured.given;
+import static json_homework.json_reader.JsonReader.convertJsonObjectToUrlEncoded;
+import static org.hamcrest.Matchers.equalTo;
+import static petstore.PetstoreConstants.BASE_URL;
+import static petstore.pet.constants.PetConstants.*;
+
+public class PetApiTest {
+    private static final String PET_ENDPOINT = StringUtils.join(BASE_URL, ADD_UPDATE_PET_URL);
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:23]: Make tests independent from each other

File: src/test/java/petstore/pet/PetApiTest.java [hunk 11]
```diff
@@ -0,0 +1,308 @@
...
+                .log()
+                .all();
+    }
+
+    /**
+     * Test the valid test cases when adding a photo to a pet
+     */
+    @Test(dataProvider = "validAddPhotoByIdData", dataProviderClass = PetApiData.class, priority = 1)
+    public void validAddPhotoByIdTest(JsonObject updateByIdData) {
+        String id = updateByIdData.get("petId").getAsString();
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:89]: Move work with json, urls and path to other place

File: src/test/java/petstore/pet/PetApiTest.java [hunk 12]
```diff
@@ -0,0 +1,308 @@
...
+    public void validDeletePetResponse(JsonObject petData) {
+        String petId = petData.get("petId").getAsString();
+        String url = StringUtils.join(PET_ENDPOINT, "/", petId);
+        Response response = given()
+                .accept(CONTENT_TYPE_JSON)
+                .when()
+                .delete(url);
+        response
+                .then()
+                .statusCode(200)
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:164]: Replace with constatns

File: src/test/java/petstore/pet/constants/PetConstants.java [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+    public static final String ADD_UPDATE_PET_URL = "/pet";
+    public static final String UPLOAD_IMAGE = "/uploadImage";
+    public static final String FIND_BY_STATUS_PET_URL = "/findByStatus?status=";
+    public static final String VALID_ADD_PET_TEMPLATE = "validAddPetBody"; // Updated
+    public static final String VALID_DELETE_PET_TEMPLATE = "validDeletePetBody"; // Updated
+    public static final String VALID_SEARCH_BY_STATUS_PET_TEMPLATE = "validFindByStatus"; // Updated
+    public static final String VALID_UPDATE_BY_ID_PET_TEMPLATE = "validUpdateById"; // Updated
+    public static final String VALID_UPLOAD_IMAGE_BY_ID_TEMPLATE = "validUploadImageById"; // Updated
+    public static final String VALID_UPLOAD_IMAGE_BY_ID_DATA = "validUploadImageByIdData"; // Updated
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/constants/PetConstants.java:17]: Divide into sections request/response/test data

File: src/test/java/petstore/pet/data/PetApiData.java [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+    public Object[][] validUpdatePetTestData() throws IOException {
+        JsonObject[] updatePetData = readTestDataAndBuild(PET_BODY_TEST_DATA, PET_BODY_TEMPLATE, VALID_ADD_PET_TEMPLATE, VALID_UPDATE_PET_TEST_DATA);
+        return mapTestData(updatePetData);
+    }
+    /**
+     *Retrieve the valid data to find a pet
+     */
+    @DataProvider(name = "validFindPetTestData")
+    public Object[][] validFindPetTestData() throws IOException {
+        JsonObject[] findPetData = readTestDataAndBuild(PET_BODY_TEST_DATA, PET_BODY_TEMPLATE, VALID_DELETE_PET_TEMPLATE, VALID_DELETE_PET_TEST_DATA);
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/data/PetApiData.java:36]: You can avoid creating a findPetData variable

File: src/test/java/petstore/store/constants/storeApiDataTest.json [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+      "petId": 1,
+      "quantity": 10,
+      "shipDate": "2023-10-05T09:15:20.123Z",
+      "status": "sold",
+      "complete": true
+    }
+  ],
+  "validFindOrder": [
+    {
+      "order": "/2147473647"
```
Comment (BohdanKalep) [inline src/test/java/petstore/store/constants/storeApiDataTest.json:22]: reuse oreder with %s

File: src/test/java/petstore/store/constants/storeApiDataTest.json [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+    {
+      "id": 2147473647,
+      "petId": 1,
+      "quantity": 1,
+      "shipDate": "2023-10-05T09:15:20.123Z",
+      "status": "sold",
+      "complete": true
+    },
+    {
+      "id": 2147473648,
```
Comment (BohdanKalep) [inline src/test/java/petstore/store/constants/storeApiDataTest.json:12]: get Id from the response and work with it

File: src/test/resources/testng.xml [hunk 17]
```diff
@@ -10,5 +10,17 @@
             </class>
         </classes>
     </test>
+    <test name="Petstore Test">
+        <classes>
+            <class name="petstore.pet.PetApiTest">
+            </class>
```
Comment (BohdanKalep) [inline src/test/resources/testng.xml:16]: You can close tag like this \> instead writing  </class>

**Review by BohdanKalep — CHANGES_REQUESTED** [review]

File: src/test/java/api_constants/ApiTestConstants.java [hunk 18]
```diff
@@ -0,0 +1,9 @@
+package api_constants;
+
+public class ApiTestConstants {
+    public static final String CODE = "code";
+    public static final String EMPTY_ARRAY = "[]";
+
+
```
Comment (BohdanKalep) [inline src/test/java/api_constants/ApiTestConstants.java:7]: Remove extra lines in this class

File: src/test/java/petstore/constants/PetStoreConstantsConfig.properties [hunk 19]
```diff
@@ -0,0 +1,22 @@
+template.user=userTemplate
+template.order=order
+template.validFindByStatus=validFindByStatus
+testData.validFindByStatus=validFindByStatusTestData
+testData.invalidFindByStatus=invalidFindByStatusTestData
+constants.defaultId=0
+paths.storeApiTemplate=src/test/java/petstore/store/constants/storeApiTemplate.json
+paths.petBodyTemplate=src/test/java/petstore/pet/constants/petApiTemplate.json
+paths.userApiTemplate=src/test/java/petstore/user/constants/userApiTemplate.json
+paths.petBodyTestData=src/test/java/petstore/pet/constants/petApiDataTest.json
```
Comment (BohdanKalep) [inline src/test/java/petstore/constants/PetStoreConstantsConfig.properties:10]: Please keep paths for files and urls separate

File: src/test/java/petstore/pet/PetApiTest.java [hunk 20]
```diff
@@ -0,0 +1,266 @@
...
+    @Test(dataProvider = "validPetData", dataProviderClass = PetApiData.class)
+    public void validUpdatePetTest(JsonObject petData) {
+        Response addPet = getAddPet(petData);
+        JsonObject responseBody = JsonParser.parseString(addPet.getBody().asString()).getAsJsonObject();
+        responseBody = refillRandomValidPet(responseBody);
+        Response updatePet = requestPutJSON(responseBody, PET_ENDPOINT);
+        logResponseDetails(
+                assertOk(updatePet)
+                        .body(CODE, Matchers.not(equalTo(SC_NOT_FOUND))));
+
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:75]: Please remove extra lines here

File: src/test/java/petstore/pet/PetApiTest.java [hunk 21]
```diff
@@ -0,0 +1,266 @@
...
+    public void validFindPetTest(JsonObject petFindData) {
+        Response createPet = getAddPet(petFindData);
+        JsonObject responseBody = JsonParser.parseString(createPet.getBody().asString()).getAsJsonObject();
+        Response findPet =
+                requestGet(
+                        String.format(PET_ID_URL, responseBody.get("id").getAsString()));
+        logResponseDetails(
+                assertOk(findPet)
+                        .body(CODE, Matchers.not(equalTo(SC_NOT_FOUND))));
+
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:93]: Please remove extra lines here

File: src/test/java/petstore/pet/PetApiTest.java [hunk 22]
```diff
@@ -0,0 +1,266 @@
...
+        deletePet(createPet);
+    }
+
+    /**
+     * Test the valid test cases when updating a pet by id
+     */
+    @Test(dataProvider = "validPetData", dataProviderClass = PetApiData.class)
+    public void validUpdateByIdData(JsonObject updateByIdData) {
+        Response createPetResponse = getAddPet(updateByIdData);
+        JsonObject responseBody = createPetResponse.jsonPath().getObject("", JsonObject.class);
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:121]: Use StringUtils.EMPTY instead of ""

File: src/test/java/petstore/pet/PetApiTest.java [hunk 23]
```diff
@@ -0,0 +1,266 @@
...
+
+        responseBody = refillRandomValidPet(responseBody);
+
+        Response updateData = requestPostURLENC(updateByIdData,
+                String.format(PET_ID_URL, responseBody.get("id").getAsString()));
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:126]: as long as you repeating id several times create a constant

File: src/test/java/petstore/pet/PetApiTest.java [hunk 24]
```diff
@@ -0,0 +1,266 @@
...
+    /**
+     * Test the valid test cases when finding a pet
+     */
+    @Test(dataProvider = "validPetData", dataProviderClass = PetApiData.class)
+    public void validFindPetTest(JsonObject petFindData) {
+        Response createPet = getAddPet(petFindData);
+        JsonObject responseBody = JsonParser.parseString(createPet.getBody().asString()).getAsJsonObject();
+        Response findPet =
+                requestGet(
+                        String.format(PET_ID_URL, responseBody.get("id").getAsString()));
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:89]: create a constatnt for id

File: src/test/java/petstore/pet/PetApiTest.java [hunk 25]
```diff
@@ -0,0 +1,266 @@
...
+
+    }
+
+    /**
+     * Test the valid test cases when adding a photo to a pet
+     */
+    @Test(dataProvider = "validPetData", dataProviderClass = PetApiData.class)
+    public void validAddPhotoByIdTest(JsonObject updateByIdData) {
+        Response createPet = getAddPet(updateByIdData);
+        String id = createPet.jsonPath().getString("id");
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:102]: constant for id

File: src/test/java/petstore/pet/PetApiTest.java [hunk 26]
```diff
@@ -0,0 +1,266 @@
...
+    }
+
+    /**
+     * Test the valid test cases when deleting a pet
+     */
+    @Test(dataProvider = "validPetData", dataProviderClass = PetApiData.class)
+    public void validDeletePetResponse(JsonObject petData) {
+        Response response = getAddPet(petData);
+        deletePet(response);
+
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:156]: Please remove extra lines here

File: src/test/java/petstore/store/data/StoreApiData.java [hunk 27]
```diff
@@ -0,0 +1,65 @@
...
+public class StoreApiData {
+
+
+    public JsonObject getStoreTemplate() {
+        return getJsonTemplate(STORE_API_TEMPLATE, ORDER_TEMPLATE);
+    }
+
+    @DataProvider(name = "validOrderData")
+    public Object[][] validOrderData() {
+        int numberOfDataPoints = new Random().nextInt(2) + 2;
```
Comment (BohdanKalep) [inline src/test/java/petstore/store/data/StoreApiData.java:27]: Avoid usage of magic numbers create a variable and put in the variable name purpuse of the value

File: src/test/java/petstore/store/data/StoreApiData.java [hunk 28]
```diff
@@ -0,0 +1,65 @@
...
+
+        return IntStream.range(0, numberOfDataPoints)
+                .mapToObj(i -> new Object[]{fillRandomValidOrder(getStoreTemplate())})
+                .toArray(Object[][]::new);
+    }
+
+    @DataProvider(name = "invalidOrderData")
+    public Object[][] invalidOrderData() {
+        int numberOfDataPoints = new Random().nextInt(7) + 1; // Generate a random number between 1 and 7
```
Comment (BohdanKalep) [inline src/test/java/petstore/store/data/StoreApiData.java:36]: Avoid usage of magic numbers create a variable and put in the variable name purpuse of the value

File: src/test/java/petstore/store/data/StoreApiData.java [hunk 29]
```diff
@@ -0,0 +1,65 @@
...
+
+        return IntStream.range(0, numberOfDataPoints)
+                .mapToObj(i -> new Object[]{fillRandomInvalidOrder(getStoreTemplate())})
+                .toArray(Object[][]::new);
+    }
+
+
+    public static JsonObject fillRandomValidOrder(JsonObject jsonTemplate) {
+        String petId = String.valueOf(new Random().nextInt(100) + 1);
```
Comment (BohdanKalep) [inline src/test/java/petstore/store/data/StoreApiData.java:45]: Avoid usage of magic numbers create a variable and put in the variable name purpuse of the value

File: src/test/java/petstore/store/data/StoreApiData.java [hunk 30]
```diff
@@ -0,0 +1,65 @@
...
+        String shipDate = generateActualDate(); // Random URL
+        String status = generateRandomString();
+        String complete = generateRandomBooleanString();
+        String filledJson = String.format(String.valueOf(jsonTemplate), DEFAULT_ID, petId, quantity, shipDate, status, complete);
+
+        return JsonParser.parseString(filledJson).getAsJsonObject();
+    }
+
+    public static JsonObject fillRandomInvalidOrder(JsonObject jsonTemplate) {
+        String petId = String.valueOf(new Random().nextInt(100) + 1);
```
Comment (BohdanKalep) [inline src/test/java/petstore/store/data/StoreApiData.java:56]: Avoid usage of magic numbers create a variable and put in the variable name purpuse of the value

File: src/test/java/petstore/user/data/UserApiData.java [hunk 31]
```diff
@@ -0,0 +1,123 @@
...
+import static petstore.utils.PetstorePathHandler.USER_API_TEMPLATE;
+
+public class UserApiData {
+    public JsonObject getUserTemplate() {
+        return getJsonTemplate(USER_API_TEMPLATE, USER_TEMPLATE);
+    }
+
+    @DataProvider(name = "validUserData")
+    public Object[][] validUserData() {
+        int numberOfDataPoints = new Random().nextInt(2) + 2;
```
Comment (BohdanKalep) [inline src/test/java/petstore/user/data/UserApiData.java:24]: You you same randomizer for user and for the pet. Maybe you can create a general method with parameters to use it in all necessary places.

**Review by BohdanKalep — CHANGES_REQUESTED** [review]
UserApiData/ PetApiData / StoreApiData
they used a lot of repetative String initialisations.
It would be better to create a DTO. A class with all necessary fileds, status,id,name and after call this clas and use a setter to set a proper name. In this case you will not need to create the same variables in each method with different values

File: src/test/java/petstore/pet/data/PetApiData.java [hunk 32]
```diff
@@ -0,0 +1,145 @@
...
+        int randomIndex = new Random().nextInt(images.size());
+        String imagePath = StringUtils.join(
+                images.get(randomIndex).getAsJsonObject().getAsJsonPrimitive("file").getAsString());
+        image.addProperty("file", imagePath);
+        return image;
+    }
+
+    private static JsonObject fillCategory() {
+        JsonObject category = new JsonObject();
+        category.addProperty("id", DEFAULT_ID);
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/data/PetApiData.java:83]: change string values for constans

File: src/test/java/petstore/pet/data/PetApiData.java [hunk 33]
```diff
@@ -0,0 +1,145 @@
...
+        category.addProperty("name", generateRandomString());
+        return category;
+    }
+
+    public static JsonObject refillRandomValidPet(JsonObject randomPet) {
+        JsonObject newPet = new JsonObject();
+        newPet.addProperty("id", randomPet.get("id").getAsString());
+        newPet.add("category", fillCategory());
+        newPet.addProperty("name", generateRandomString());
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/data/PetApiData.java:92]: change string values for constans

File: src/test/java/petstore/pet/data/PetApiData.java [hunk 34]
```diff
@@ -0,0 +1,145 @@
...
+
+    public static JsonObject refillRandomInvalidPet(JsonObject randomPet) {
+        JsonObject newPet = new JsonObject();
+        newPet.addProperty("id", randomPet.get("id").getAsString());
+        JsonObject category = new JsonObject();
+        category.addProperty("id", generateRandomString());
+        category.addProperty("name", generateRandomString());
+        newPet.add("category", category);
+        newPet.addProperty("name", generateRandomString());
+        newPet.add("photoUrls", generateRandomPhotoUrls());
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/data/PetApiData.java:107]: change string values for constans

File: src/test/java/petstore/store/StoreApiTest.java [hunk 35]
```diff
@@ -0,0 +1,120 @@
...
+
+    private static void validOrderDelete(Response addOrder) {
+        String id = getId(addOrder);
+        Response deleteOrder = requestDelete(StringUtils.join(ORDER_URL, "/", id));
+        logResponseDetails(assertOk(deleteOrder));
+    }
+
+    private static String getId(Response addOrder) {
+        JsonObject responseBody = JsonParser.parseString(addOrder.getBody().asString()).getAsJsonObject();
+        return responseBody.get("id").getAsString();
```
Comment (BohdanKalep) [inline src/test/java/petstore/store/StoreApiTest.java:70]: change id string values for constans

File: src/test/java/petstore/user/data/UserDTO.java [hunk 36]
```diff
@@ -0,0 +1,78 @@
+package petstore.user.data;
+
+import java.io.Serializable;
+
+public class UserDTO implements Serializable {
```
Comment (BohdanKalep) [inline src/test/java/petstore/user/data/UserDTO.java:5]: Can you explain why do you need Serializable interface here?

File: src/test/java/petstore/pet/PetApiTest.java [hunk 37]
```diff
@@ -0,0 +1,262 @@
...
+        deletePet(addPet);
+    }
+
+    /**
+     * Test the invalid test cases when searching pets by status
+     */
+    @Test(dataProvider = "invalidSearchByStatusData", dataProviderClass = PetApiData.class)
+    public void invalidSearchByStatusTest(JsonObject updateByIdData) {
+        String status = updateByIdData.get("status").getAsString();
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:236]: change string values for constans

File: src/test/java/petstore/pet/PetApiTest.java [hunk 38]
```diff
@@ -0,0 +1,262 @@
...
+
+        deletePet(createPetResponse);
+    }
+
+    /**
+     * Test the valid test cases when searching pets by status
+     */
+    @Test(dataProvider = "validSearchByStatusData", dataProviderClass = PetApiData.class)
+    public void validSearchByStatusTest(JsonObject updateByIdData) {
+        String status = updateByIdData.get("status").getAsString();
```
Comment (BohdanKalep) [inline src/test/java/petstore/pet/PetApiTest.java:137]: change string values for constans

File: pom.xml [hunk 39]
```diff
@@ -55,7 +87,12 @@
                     </suiteXmlFiles>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.qameta.allure</groupId>
+                <artifactId>allure-junit4</artifactId>
```
Comment (BohdanKalep) [inline pom.xml:92]: You wanted to use allure with this homework as well? Or pushed it accidently?
Make sure you use dependence for testNg as long as you use this framework and not junit

Comment (CarlosGuadamuz0) [reply pom.xml]: Sorry it was an error

**Review by BohdanKalep — APPROVED** [review]

**Pull request merged**